src/components/carousel/index.jsx

```diff
@@ -115,6 +142,12 @@ const Carousel = ({
 		setPosition(position - 100 / slidesToShow);
 	};
 
+	// a prefers-reduced-motion user setting must always override Autoplay
+	const AutoplayEnabledAndAllowed =
+		enableAutoplay && !!window.matchMedia("'(prefers-reduced-motion: reduce)");
```

Comment:
`window` will throw an error on Fusion server side render and cause this code to become client side only. We should add an `isServerSide()` util function to handle checking for `window` being defined

Reply:
@matthewroach updated. thanks for the note